feat(transition_tool): Use execution-specs daemon (#453)
ethereum/execution-spec-tests: docs/CHANGELOG.md

### 🔧 EVM Tools

> - ✨ Use `execution-specs` evm tool daemon feature to run tests, which provides an fill speed increase of around 57x ([#453](https://github.com/ethereum/execution-spec-tests/pull/453))

Reviewer: ```suggestion
- ✨ Use `execution-specs` evm tool daemon feature to fill tests, which provides a speed increase of around 57x ([#453](https://github.com/ethereum/execution-spec-tests/pull/453))[[
```
